cmd/secret/cmd/root.go

```diff
@@ -115,7 +117,7 @@ func (r *root) setCmd() *cobra.Command {
 		Long:  "it takes keys and a value from user and saves value in encrypted manner in specified storage",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			var ds secretApi.DataSaver
-			var cr = crypto.NewCryptographer([]byte(cipherKey))
+			var cr = crypto.NewCryptographer([]byte(cipherKey), cryptoReader.NewRootReader([]byte(cipherCode)))
```

Comment:
Let's making even better: to prevent user with the same `cipher-key` to provide different codes for different keys we simplify this process:
- take hash from cipherKey
- provide this hash as a code into the reader.

That way we preserve the next: the same cipher key will have its unique code that allows to find keys in a storage in repeatable manner.

No need in the flag `--cipher-code` then.

------
To make testing easier create a field in `root` structure that will have `nonceCode io.Reader`. It could be used as a substitution for a calculated code if provided.